about.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="icon" href="./images/gc-logo.png" type="image/x-icon">
+  <link href="//fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="//fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <title>Gaming Convention 2022</title>
+</head>
+
+<body>
+  <header></header>
+
+  <section class="headline">
+    <h1 class="page-header">GAMING CONVENTION 2022 IS HERE!</h1>
+    <hr class="orange-hr">
+    <p class="about-convention">The Gaming Convention brings together the community of gamers, creators, streamers, game
+      designers and many more from all over the world every two years. <br> Gaming Convention are a gathering
+      centered on role-playing games, collectible card games, miniatures wargames, board games, video games, or other
+      types of games. <br> These conventions are typically two or three days long, and often held at either a university
+      or in a convention center hotel. <br> Gaming conventions typically have games scheduled for various time slots.
+      Most gaming cons use some system of pre-registration to ensure that games are neither over-full nor under-full.
+    </p>
+    <p class="contact"> <span id="contact-text">Please contact as by e-mail if you have any other questions about the
+        event</span> <br> <span id="email">contact@gamingconvention.com</span></p>
+  </section>
+
+  <section id="logo-section">
+    <h2 class="section-header">Gaming Convention 2022 logo</h2>
+    <hr class="orange-hr">
+    <p class="sub-text">The logo of GC (Gaming Convention) was created by Manel Hammouche on April, 2022 using
+      "logopit" (a mobile application)</p>
+    <img src="./images/gc-logo.png" alt="" id="about-logo">
+  </section>
+
+  <section id="past-events">
+    <h2 class="section-header">See the past Gaming Conventions</h2>
+    <hr class="orange-hr">
+    <p class="sub-text">Take a look at the last two Gaming Conventions which took place in Tokyo and NewYork</p>
+    <div id="past-events-imgs">
+      <a href="#" class="event" id="event-1"><span>2019</span><span>GC 2019 in Tokyo</span></a>
+      <a href="#" class="event" id="event-2"><span>2017</span><span>GC 2019 in NewYork</span></a>
+    </div>
+  </section>
+
+  <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+  <script src="./scripts/page-header.js"></script>
+  <script src="./scripts/sponsors-footer.js"></script>
```

Comment:
- [ ] Kindly change the footer color in the desktop version as it's light in the Behance design on the home page.